draft-ietf-rats-corim.md

```diff
@@ -2351,6 +2425,21 @@ For each `rv` entry, the `condition` ECT is compared with an ACS ECT, where the
 
 If the ECTs match except for authority, the `rv` `addition` ECT authority is added to the ACS ECT authority.
 
+#### Processing Domain Membership {#sec-process-dm}
+
+Domain Membership Triples allow an Endorser (for example an Integrator) to issue an authoritative statement about the composition of an Attester as a collection of Environments.
+If the Verifier Appraisal policy requires Domain Membership, then membership triple sets a reference composition of an Attester in a Verifier database.
```

Comment:
What about the policy requiring domain membership has to do with the interpretation of the membership triple?